common/src/main/java/gov/cms/ab2d/common/model/Job.java

```diff
@@ -25,6 +19,7 @@
 @Getter
 @Setter
 @EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Configurable(autowire = Autowire.BY_TYPE)
```

Comment:
This seems strange to me. 
IMHO, an entity should almost never be polluted with spring package names.
Spring beans are singleton unlike entities.
I am not sure what is being autowired here, but I do not think that entities should be aware of Spring (or any dependency injection framework used).

Reply:
I was playing around with trying to put in some validation here, I forgot to remove this.